src/hooks/useFetchListData.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState, useRef } from "react";
+import { useRouter } from "next/router";
+import { sleep } from "shared/sleep";
+
+interface FetchDataParams {
+  addressHash: string;
+  triggerApiCall: () => any;
```

Comment:
```suggestion
  triggerApiCall: () => void;
```

Reply:
will get this error 😞 
<img width="749" alt="image" src="https://github.com/BirthdayResearch/metascan/assets/17063281/46231034-5ad0-4e2e-a72d-e8121a262959">